### ISSUE TITLE
make the whisper bubble noise fall off scale with whisper max distance

### ISSUE THREAD

**Issue by epicEaston197** [Neos-Metaverse/NeosPublic#3777]
### Is your tweak request related to a problem? Please describe.

the problem here is I can make my whisper bubble bigger with whisper Max distance and people can still hear me even with the enlarged whisper bubble but the noise fall off does not scale with the new whisper bubble size which means I cannot hear people if they are not close to me inside of my whisper bubble

### Describe what would you like tweaked

I would like the whisper bubble noise fall off to scale with whisper max distance

### Describe alternatives you've considered

unfortunately the whisper bubble noise fall off is hard-coded I believe so there is no alternative but if there is one at please tell me

**Comment by ProbablePrime** [comment]
I think this is a duplicate of: https://github.com/Neos-Metaverse/NeosPublic/issues/1195

Can you check Easton?

**Comment by epicEaston197** [comment]
> I think this is a duplicate of: #1195
> 
> Can you check Easton?

it sounds really similar but I think it's different this issue is relating to the noise fall off from inside of the whisper bubble

**Comment by LexiBasilisk** [comment]
I read through your description and I also believe it is the same issue, @epicEaston197.

Do you have any additional detail you can provide? The way the whisper bubble works is that it's a hard cutoff based on distance- either you're in range, or you're out of range.

The issue is that despite a user visually being in your bubble, you aren't within the range of theirs hence why you can't hear them.

**Comment by epicEaston197** [comment]
> I read through your description and I also believe it is the same issue, @epicEaston197.
> 
> Do you have any additional detail you can provide? The way the whisper bubble works is that it's a hard cutoff based on distance- either you're in range, or you're out of range.
> 
> The issue is that despite a user visually being in your bubble, you aren't within the range of theirs hence why you can't hear them.

this has nothing to do with other people's whisper bubbles this issue is only for the audio fall-off to scale with max distance

**Comment by LexiBasilisk** [comment]
So to clarify then- are you saying that if two separate users set the same increased max distance, and they were both visually within range of each other's bubbles, that there would still be a dead-zone where they can't hear each other?

I'm not quite sure what you mean by 'audio fall-off' otherwise? If you're referring to the volume of users that are outside of the whisper- that is controlled by a setting you can adjust in the settings menu.

If I'm still not understanding something, do you think you could grab a friend, and record a demonstration of the issue? I'm apparently still not understanding what your issue is, if it's not anything I've said above or previously.

**Comment by epicEaston197** [comment]
> So to clarify then- are you saying that if two separate users set the same increased max distance, and they were both visually within range of each other's bubbles, that there would still be a dead-zone where they can't hear each other?
> 
> I'm not quite sure what you mean by 'audio fall-off' otherwise? If you're referring to the volume of users that are outside of the whisper- that is controlled by a setting you can adjust in the settings menu.
> 
> If I'm still not understanding something, do you think you could grab a friend, and record a demonstration of the issue? I'm apparently still not understanding what your issue is, if it's not anything I've said above or previously.

yes that there would be a dead zone because of the hard-coded fall off what I mean by fall off is when you are in a whisper bubble of default scale when somebody comes into your whisper bubble they enter the audio fall off

 and when they walk out there audio gets lowered but when you make the bubble bigger the fall-off is the same resulting in the user visually being within range of your bubble but your in able to hear them
 
 I think both of our issues are stemmed from the same problem
